common/src/main/java/com/redhat/cloud/notifications/models/IntegrationTemplate.java

```diff
@@ -36,10 +36,14 @@ public class IntegrationTemplate extends CreationUpdateTimestamped {
     @JsonProperty(access = READ_ONLY)
     private UUID id;
 
-    @OneToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
If `applicationId` is not used while dealing with event type template (as I suggested in other comments), then I wonder if that change is needed... :thinking:

Reply:
Because we can have one template per event type, and we can have multiple event types for a single application, this is why it seems natural for me to update it to have a consistent data on this table.